end-to-end-tests/specs/screenshot.spec.js

```diff
@@ -218,7 +218,6 @@ describe('patient view page screenshot test', function(){
     before(function(){
         var url = `${CBIOPORTAL_URL}/patient?studyId=lgg_ucsf_2014&caseId=P04`;
         goToUrlAndSetLocalStorage(url);
-        browser.setViewportSize({ height: 1600, width: 1600 });
```

Comment:
we need to leave this one because otherwise the genomic overview doesnt expand correctly